tools/gen-wire-tests/main.go

```diff
@@ -491,6 +509,23 @@ const Template = `
           fail: true
           type: absolute
     builders:
+{{- if index $.MinVersions $task_name }}
+      - conditional-step:
+          condition-kind: regex-match
+          regex: "{{ index $.MinVersions $task_name }}"
+          label: "$JUJU_VERSION"
+          steps:
+            - {{$builder}}:
+                  test_name: '{{$.SuiteName}}'
+                  setup_steps: ''
+{{- if gt (len $node.SkipTasks) 1 }}
```

Comment:
can this macro IFs be indented so it's more explicit that they are nested?

Reply:
Addressed in new commit